### PR TITLE
Ptt/init reporting

### PR DESCRIPTION
**Pull request by avtoku**
Changes have been made to allow the board to send initialization status messages to the companion computer.
This requires also updating the board supports package in the varmint_h7 directory.

### CODE REVIEW

File: include/comm_manager.h [hunk 1]
```diff
@@ -180,8 +180,9 @@ class CommManager : public CommLinkInterface::ListenerInterface, public ParamLis
   void send_param_value(uint16_t param_id);
   void update_status();
   void log(CommLinkInterface::LogSeverity severity, const char * fmt, ...);
+  void log_message(CommLinkInterface::LogSeverity severity, char * text);
 
-  void send_parameter_list();
+//  void send_parameter_list();
```
Comment (bsutherland333) [inline include/comm_manager.h:185]: It would be better to remove old code than comment it out

File: src/comm_manager.cpp [hunk 2]
```diff
@@ -129,7 +129,10 @@ void CommManager::param_request_list_callback(uint8_t target_system)
   if (target_system == sysid_) { send_params_index_ = 0; }
 }
 
-void CommManager::send_parameter_list() { send_params_index_ = 0; }
+//void CommManager::send_parameter_list()
```
Comment (bsutherland333) [inline src/comm_manager.cpp:132]: Same as above

File: src/comm_manager.cpp [hunk 3]
```diff
@@ -526,6 +534,23 @@ void CommManager::send_named_value_float(const char * const name, float value)
 
 void CommManager::send_next_param(void)
 {
+  //PTT Hack
```
Comment (bsutherland333) [inline src/comm_manager.cpp:537]: Should we figure out a more permanent solution for this before merging these changes into main?

**Review by bsutherland333 — CHANGES_REQUESTED** [review]

File: include/comm_manager.h (hunk 1 above)
Comment (avtoku) [reply include/comm_manager.h]: done

File: src/comm_manager.cpp (hunk 2 above)
Comment (avtoku) [reply src/comm_manager.cpp]: done

File: src/comm_manager.cpp (hunk 3 above)
Comment (avtoku) [reply src/comm_manager.cpp]: yes we should think of a more permanent solution

**Pull request merged**